### PR TITLE
Add Hungarian translation

### PR DESCRIPTION
**Pull request by nagyrobi**
Please add also the sensor names to the strings. Like `Program phase`, `Status`, `Elapsed time`, `Finish at`, etc. Also we need translation for `min` (minutes).

### CODE REVIEW

**Comment by astrandb** [comment]
> Please add also the sensor names to the strings. Like `Program phase`, `Status`, `Elapsed time`, `Finish at`, etc. Also we need translation for `min` (minutes).

There is currently no support for localization of sensor names in Home Assistant. I know that people are working to make it possible and as soon as this feature is released (and documented) we will try to implement it.

File: custom_components/miele/translations/hu.json [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+          "white_tea": "Tea",
+          "japanese_tea": "Japán tea",
+          "green_tea": "Zöld tea",
+          "fruit_tea": "Gyümölcs tea",
+          "herbal_tea": "Herbál tea",
+          "appliance_rinse": "Öblítés",
+          "milk_pipework_rinse": "Tejcső öblítés",
+          "milk_pipework_clean": "Tejcső tisztítás",
+          "brewing_unit_degrease": "Főzőegység zsírtalanítás",
+          "descaling": "Vízkőmentesítés"
```
Comment (astrandb) [inline custom_components/miele/translations/hu.json:160]:
```suggestion
          "descaling": "Vízkőmentesítés",
          "appliance_settings": "Appliance settings menu"   
```

Comment (astrandb) [reply custom_components/miele/translations/hu.json:160]: Please add translation for this recently added mapping for the coffee maker also.

**Review by astrandb — CHANGES_REQUESTED** [review]
Looks goos, thank you.

File: custom_components/miele/translations/hu.json (hunk 1 above)
Comment (nagyrobi) [reply custom_components/miele/translations/hu.json]: Added.

**Review by astrandb — APPROVED** [review]

**Pull request merged**